frontend/www/js/omegaup/components/course/FilteredList.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="panel-body tab-container">
+    <ul class="nav nav-tabs">
+      <li
+        class="nav-item"
+        v-bind:class="{
+          active: activeTab === `${type}-courses-${filteredCourses.type}`,
```

Comment:
aquí no es suficiente hacer

```suggestion
          active: activeTab === filteredCourses.type,
```

? `-courses-` está de más porque este componente es una lista sólo de cursos, y el otro `${type}` no se qué sea.

Reply:
Oh, ya entendí. Al sub-componente ya no le importa si pertenece a estudiante o a admin, así que `type` ya no es necesario. Se quita